### DISCUSSION TITLE
Specifying DefaultBatchFormatter in settings.json

### DISCUSSION THREAD

**Discussion by pankaj-k** [FantasticFiasco/serilog-sinks-http#200] [General]
Hi,

Is it possible to specify in settings.json a new value for DefaultBatchFormatter so that it will not drop log events that exceed 256KB? 
I was looking for some sample codes but was not able to find any.

Regards

**Comment by FantasticFiasco** [comment]
Yes, you can specify a value in the settings file, but not one that would automatically be picked up by the sink. What you can do is to define the setting, and then pull it yourself and inject it into the `DefaultBatchFormatter`.

**Reply by pankaj-k** [reply]
Thanks for reply. I was trying to see if it can be done settings.json itself. It is easier to change the file in systems deployed in production.

**Reply by FantasticFiasco** [reply]
Sorry for the confusion I seem to have introduced, but I also propose to add the setting to `settings.json`, the same file you store your Serilog settings in, and then pull the value and inject it into the batch formatter.
